SU-SWS/stanford_sites_helper
# DRPL-743: Update links to SNOW forms

# READY FOR REVIEW

# Summary
- The links to our SNOW forms were incorrect.

# Needed By (Date)
- Wednesday, August 15th.

# Criticality
- How critical is this PR on a 1-10 scale? 3/10

# Steps to Test

1. Go to https://sites-uat.stanford.edu/drupal7-update-status/admin/config/stanford-sites-helper/filequota/check
2. Click the help ticket links
3. Ensure that they go to the correct SNOW forms:
    1. "Problems using this service..." link should go to https://stanford.service-now.com/it_services?id=sc_cat_item&sys_id=abff87fa4f33f2009a9a97411310c723
    2. "Please submit a help ticket for more storage" should go to https://stanford.service-now.com/it_services?id=sc_cat_item&sys_id=273aa1874f0b83009a9a97411310c796

# Associated Issues and/or People
- JIRA ticket: https://asjira.stanford.edu/jira/browse/DRPL-743

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
